library/dnet.c

```diff
@@ -319,7 +319,7 @@ static void dnet_queue_wait_threshold(struct dnet_net_state *st)
 	/* If send succeeded then we should increase queue size */
 	while ((atomic_read(&st->send_queue_size) > DNET_SEND_WATERMARK_HIGH) && !st->__need_exit) {
 		/* If high watermark is reached we should sleep */
-		dnet_log(st->n, DNET_LOG_DEBUG,
+		dnet_log(st->n, DNET_LOG_ERROR,
```

Comment:
I still don't see an error here, so I prefer using NOTICE/INFO level because on fast device it will spam logs without any help. It is better to add logs around similar threshold in server-send.